backend/src/middleware/email/templates/passwordReset.js

```diff
@@ -53,6 +54,7 @@ export const wrongAccountMail = options => {
   } = options
   const actionUrl = new URL('/password-reset/request', CONFIG.CLIENT_URI)
   return {
+    from: '"Human Connection" <info@human-connection.org>',
```

Comment:
We already forgot to set this. Can we store this somewhere at a central location, so we never forget to add it? Seeing it twice in the same file makes me nervous.

Reply:
Well, it feels more natural to have it there. Maybe it should be the task of QR to not have it slip through. :thinking:

Reply:
I also felt it made more sense to place the `from` alongside the other attributes. But I'm hoping to refactor the passwordReset in such a way that I can get rid of the duplicate!